sources/packages/backend/libs/test-utils/src/factories/application-exception-request.ts

```diff
@@ -15,16 +16,23 @@ import { createFakeApplicationException } from "./application-exception";
  */
 export function createFakeApplicationExceptionRequest(relations: {
   applicationException?: ApplicationException;
-  creator: User;
+  creator?: User;
 }): ApplicationExceptionRequest {
   const applicationExceptionRequest = new ApplicationExceptionRequest();
-  applicationExceptionRequest.applicationException =
-    relations.applicationException ??
-    createFakeApplicationException({
-      applicationExceptionStatus: ApplicationExceptionStatus.Pending,
-      creator: relations.creator,
-    });
-  applicationExceptionRequest.exceptionName = relations.creator.firstName;
+
+  if (relations.applicationException) {
+    applicationExceptionRequest.applicationException =
+      relations.applicationException;
+  } else {
+    applicationExceptionRequest.applicationException =
+      createFakeApplicationException({
+        creator: relations.creator,
+      });
+    applicationExceptionRequest.applicationException.exceptionStatus =
```

Comment:
Just asking, why is a concern from this factory to set the `exceptionStatus` as `ApplicationExceptionStatus.Pending`?